docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -6,14 +6,253 @@ The parent key for all of the following parameters is `lieutenant`.
+
+
+== `operator.image`
+
+[horizontal]
+type:: string
+default:: `docker.io/projectsyn/lieutenant-operator:v0.5.3`
+
+The lieutenant-operator container image to deploy.
+
+
+== `operator.common_labels`
+
+[horizontal]
+type:: dict
+default: ::
 [source,yaml]
 ----
-namespace: example-namespace
+app.kubernetes.io/name: 'lieutenant-operator'
+app.kubernetes.io/part-of: 'project-syn'
 ----
+
+Common label to add to each resource of the lieutenant-operator. Will be used
+as a label selector for pods.
+
+
+== `operator.default_deletion_policy`
+
+[horizontal]
+type:: string
+default:: `Archive`
+
+Sets what deletion policy for external resources (Git, Vault) should be used by default.
+One of `Archive`, `Delete`, `Retain`.
+
+
+== `operator.deletion_protection`
+
+[horizontal]
+type:: bool
+default:: `false`
+
+Defines whether the annotation to protect for accidental deletion should be set by default.
+
+== `operator.default_global_git_repo`
+
+[horizontal]
+type:: string
+default:: ``
+
+URL of the default global configuration git repository. Its value will be applied to `tenant.Spec.GlobalGitRepoURL`.
+
+== `operator.vault.enabled`
+
+[horizontal]
+type:: bool
+default:: `true`
+
+Whether to create any Vault secrets.
+
+
+== `operator.vault.addr`
+
+[horizontal]
+type:: string
+default:: `vault.todo`
+
+
+Address to the Vault instance.
+
+
+== `operator.vault.path`
+
+[horizontal]
+type:: string
+default:: `kv`
+
+The mount path of the KV secret engine to be used.
+
+
+== `api.version`
+
+[horizontal]
+type:: string
+default:: `v0.6.0`
+
+The specific lieutenant-api version to deploy.
+
+
+
+== `api.image`
+
+[horizontal]
+type:: string
+default:: `docker.io/projectsyn/lieutenant-api:v0.6.0`
+
+The lieutenant-api container image to deploy.
+
+
+== `api.common_labels`
+
+[horizontal]
+type:: dict
+default: ::
+[source,yaml]
+----
+app.kubernetes.io/name: 'lieutenant-api'
+app.kubernetes.io/part-of: 'project-syn'
+----
+
+Common label to add to each resource of the lieutenant-api. Will be used
+as a label selector for pods.
+
+
+
+== `api.default_githost`
+
+[horizontal]
+type:: string
+default:: ``
+
+
+The githost to be used by default for new tenants.
+
+== `api.steward_image`
+
+[horizontal]
+type:: string
+default:: `docker.io/projectsyn/steward:v0.3.1`
+
+Image to use in generated `Steward` deployment manifests.
+
+
+
+== `api.ingress.host`
+
+[horizontal]
+type:: string
+default:: `lieutenant.todo`
+
+Defines the FQDN of the API ingress, should be overwritten on the cluster level.
+
+
+== `api.ingress.annotations`
+
+[horizontal]
+type:: dict
+default:: {}
+
+The annotations added to the created ingress. Needs to be set according to the
+deployed ingress controller.
+
+
+[source,yaml]
+----
+annotations:
+  cert-manager.io/cluster-issuer: letsencrypt-production
+  kubernetes.io/ingress.class: nginx
+  nginx.ingress.kubernetes.io/cors-allow-credentials: 'true'
+  nginx.ingress.kubernetes.io/cors-allow-methods: GET, POST, DELETE
```

Comment:
Definitely not sufficient for the API, at least `PATCH` needs to be allowed too.

Reply:
I mean you're right, the API definitely needs `PATCH`, but this config currently runs in [production](https://git.vshn.net/syn/synfra-apps/-/blob/master/lieutenant-prod/ingress.yaml).

Reply:
well, then it's fine I guess, I didn't check the production config